projects/fsastorefrontlib/src/cms-components/checkout/components/legal/legal-checkboxes/legal-checkboxes.component.html

```diff
@@ -20,7 +20,7 @@
     <span class="checkbox-text fs-icon icon-ok"></span>
   </label>
   <div class="row mt-5">
-    <div class="col-6">
+    <div class="col-12 col-md-6 mb-3">
```

Comment:
Buttons now stacking on top of each other in mobile, as on rest of the application